docs/src/modules/components/overview/Customization.tsx

```diff
@@ -0,0 +1,389 @@
+                  Color
+                </Typography>
+                <ConfigToggleButtons
+                  selectedValue={styleConfig.color}
+                  handleValueSwitch={handleChangeColor}
+                  values={Object.keys(colorScheme).map((key) => ({
+                    key,
+                    icon: <ColorSwatch color={colorScheme[key as keyof typeof colorScheme]} />,
+                  }))}
+                />
+              </Stack>
+              <Stack spacing={{ xs: 3, md: 0 }} direction={{ xs: 'row', md: 'column' }}>
+                {styleConfig.selectedTheme !== 'md3' && (
+                  <Stack spacing={1}>
+                    <Typography variant="caption" color="text.secondary" gutterBottom>
+                      Layout
+                    </Typography>
+                    <ConfigToggleButtons
+                      selectedValue={styleConfig.layout}
+                      handleValueSwitch={handleChangeLayout}
+                      values={[
+                        {
+                          key: 'horizontal',
+                          icon: <AlignHorizontalLeftIcon />,
+                        },
+                        {
+                          key: 'vertical',
+                          icon: <AlignVerticalBottomIcon />,
+                        },
+                      ]}
+                    />
+                  </Stack>
+                )}
+                {styleConfig.selectedTheme !== 'default' && (
+                  <Stack spacing={1}>
+                    <Typography variant="caption" color="text.secondary" gutterBottom>
+                      Corners
+                    </Typography>
+                    <ConfigToggleButtons
+                      selectedValue={styleConfig.corner}
+                      handleValueSwitch={handleChangeCorner}
+                      values={[
+                        {
+                          key: 'rectangular',
+                          icon: <RectangularCornersIcon />,
+                        },
+                        {
+                          key: 'medium',
+                          icon: <MediumCornersIcon />,
+                        },
+                        {
+                          key: 'rounded',
+                          icon: <RoundedCornersIcon />,
+                        },
+                      ]}
+                    />
+                  </Stack>
+                )}
+              </Stack>
+              <Stack spacing={1}>
+                <Typography variant="caption" color="text.secondary" gutterBottom>
+                  Density
+                </Typography>
+                <ConfigToggleButtons
+                  selectedValue={styleConfig.density}
+                  handleValueSwitch={handleChangeDensity}
+                  values={[
+                    {
+                      key: 'compact',
+                      icon: <DensitySmallIcon />,
```

Comment:
```suggestion
                      icon: <DensitySmallIcon fontSize="small" />,
```

I think all of the icons could be slightly smaller to improve the balance between the icon toggle button and text toggle button sizing. What do you think?

Reply:
Makes sense, thanks :)